src/app/utils/audio.ts

```diff
@@ -0,0 +1,4 @@
+export const music = document.addEventListener("DOMContentLoaded", event => {
```

Comment:
Okay, this one is bigger. Change it to a function - playAudio(volume) and import it in app.ts file. Run this method in App.start(). Volume argument should be used to init vaudio volume. Let's say default value in app.ts should be set to 0.7
Remove event listener in that file. App.start() is triggered inside this event. Moreover you can add new method in audio.ts file such as audioSettings(volume). Then in the body of this function run playAudio(volume)